pkg/services/provisioning/provisioning_test.go

```diff
@@ -0,0 +1,59 @@
+package provisioning
+
+import (
+	"context"
+	"github.com/bmizerany/assert"
```

Comment:
Used the assert here even though we do not want to keep it. I assume it will be easy to change later.

Reply:
I just merge the new lib and removed this. So please rebase your work on master so you can fix it in this PR.